contributing/community/reviews.rst

```diff
@@ -0,0 +1,190 @@
+Community Reviews
+=================
+
+Symfony is an open-source project driven by a large community. If you don't feel
+ready to contribute code or patches, reviewing issues and pull requests (PRs)
+can be a great start to get involved and give back.
+
+Why Reviewing is Important
+--------------------------
+
+Community reviews are essential for the development of the Symfony framework,
+since there are many more pull requests and bug reports than there are members
+in the Symfony core team to review, fix and merge them.
+
+On the `Symfony issue tracker`_, you can find many items in a "Needs Review"
+status:
+
+* **Bug Reports**: Bug reports primarily need to be checked for completeness.
+  Is any important information missing? Can the bug easily be reproduced?
+
+* **Pull Requests**: Pull requests contain code that fixes a bug or implements
+  new functionality. Reviews of pull requests ensure that they are implemented
+  properly, are covered by test cases, don't introduce new bugs and maintain
+  backwards compatibility.
+
+Note that **anyone who has some basic familiarity with Symfony and PHP can
+review bug reports and pull requests**. You don't need to be an expert to help.
+
+Be Constructive
+---------------
+
+Before you begin, remember that you are looking at the result of someone else's
+hard work. A good review post thanks the contributor for their work, identifies
+what was done well, identifies what should be improved, and suggests a next
+step.
+
+Create a GitHub Account
+-----------------------
+
+Symfony uses GitHub_ to manage bug reports and pull requests. If you want to
+do reviews, you need to `create a GitHub account`_ and log in.
+
+The Bug Report Review Process
+-----------------------------
+
+A good way to get started with reviewing is to pick a bug report from the
+`bug reports in need of review`_.
+
+The steps for the review are:
+
+1. **Is the Report Complete?**
+
+   Good bug reports contain a link to a fork of the `Symfony Standard Edition`_
+   (the "reproduction project") that reproduces the bug. If it doesn't, the
+   report should at least contain enough information and code samples to
+   reproduce the bug.
```

Comment:
I shared the same thought as @lsmith77 and @xabbuh, but I think that forks are really done too little, and we should experiment with trying to encourage them. Yes, it's more work for contributors, but it may also be more satisfying for them to ship some code with the bug in it, and ultimately, this is an effort to relieve burden on the core team (and a fork is the best way). Let's try it this way for now.